tdx-compliance/tdx-compliance.c

```diff
@@ -275,6 +291,10 @@ static int run_all_cpuid(void)
 
 		if (operation & 0x8000 && strcmp(case_name, t->name) != 0)
 			continue;
+		else if (operation & 0x800 && spec_version & t->version) {
+			pr_buf("%s %s\n", t->name, case_version(t->version));
+			continue;
+		}
 
 		if (!(spec_version & t->version)) continue;
```

Comment:
How about this:
```
                if (!(spec_version & t->version)) continue; 

+		if (operation & 0x800 ) {
+			pr_buf("%s %s\n", t->name, case_version(t->version));
+			continue;
+		}
```
In this case, It doesn't need to check the spec version again.